src/zcl_aes_utility.clas.abap

```diff
@@ -3,115 +3,117 @@
 *----------------------------------------------------------------------*
 *
 *----------------------------------------------------------------------*
-CLASS zcl_aes_utility DEFINITION
-  PUBLIC
-  FINAL
-  CREATE PUBLIC .
+class ZCL_AES_UTILITY definition
+  public
+  final
+  create public .
 
```

Comment:
Can you run the pretty printer for the entire class source? This will make it easier to tell what has actually been changed